packages/vanilla/package.json

```diff
@@ -9,16 +9,17 @@
     "lib/hig.css"
   ],
   "scripts": {
-    "lib": "NODE_ENV=production webpack -p",
-    "watch": "webpack --watch",
-    "deploy": "npm run lib && surge .",
-    "generate-index": "node ./dev/generate-index.js",
     "build-icons": "node src/basics/icons/build/build.js",
+    "deploy": "npm run lib && surge .",
+    "dev-server": "webpack-dev-server --open",
     "gemini": "concurrently 'http-server' 'gemini test dev/gemini/*' --kill-others",
+    "gemini-ci": "gemini test dev/gemini/*",
     "gemini-report": "concurrently 'http-server' 'gemini test dev/gemini/* --reporter html --reporter flat' --kill-others",
     "gemini-update": "concurrently 'http-server' 'gemini update dev/gemini/*' --kill-others",
-    "gemini-ci": "gemini test dev/gemini/*",
-    "test": "npm run gemini"
+    "generate-index": "node ./dev/generate-index.js",
+    "lib": "NODE_ENV=production webpack -p",
+    "test": "npm run gemini",
+    "watch": "webpack --watch"
```

Comment:
Is this vestigial now that we've got the dev server? I say kill it.

Reply:
I was hoping it would be, but this `watch` script is still needed when using `hig-react` with local `hig-vanilla` changes. I'm sure there is a clever way to consolidate, but maybe later.